udata/routing.py

```diff
@@ -59,7 +60,10 @@ class ModelConverter(BaseConverter):
     model = None
 
     def to_python(self, value):
-        obj = self.model.objects(slug=value).first()
+        try:
+            obj = self.model.objects(slug=value).first()
+        except InvalidQueryError:  # If the model doesn't have a slug.
```

Comment:
lol, I didn't kown these Python documentation links ! :)
You got the point, keep it like that :)